os_migrate/playbooks/conversion_host_content.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: conversion_hosts
```

Comment:
I don't think this is correct yet.  It's telling me the conversion host OS is Fedora.  According to the os_migration notes in gitlab, the conversion host os should be CentOS based.  Am I targeting the conversion host properly by using the hosts: block on the playbook and using the group name created in the conversion_host role?

Reply:
Yes but we'd also need to run the conversion host inventory tasks to have the `conversion_hosts` group available:

https://github.com/os-migrate/os-migrate/blob/master/os_migrate/roles/conversion_host/tasks/conv_host_inventory.yml

I think we should remove this playbook (i don't think users would typically want to try and install the content without deploying the conversion hosts first) and add this role conditionally into conversion host deployment playbook as i've suggested in the other comment. There we already have the `conversion_hosts` group available because the tasks which add it were executed earlier in the playbook.